src/server/services/feature-flags.service.ts

```diff
@@ -16,7 +16,7 @@ const featureFlags = createFeatureFlags({
   articles: ['public'],
   articleCreate: ['public'],
   adminTags: ['mod', 'granted'],
-  civitaiLink: ['mod', 'founder'],
+  civitaiLink: ['mod', 'founder', 'public'],
```

Comment:
Not sure if this should be public atm, might need confirmation from @JustMaier

Reply:
My bad. I didnt have mod/founder when in dev so I just turned it on to test. Removing now.